client/src/components/profile-page/Profile.css

```diff
@@ -0,0 +1,161 @@
+@import url("https://fonts.googleapis.com/css2?family=Cookie&display=swap");
+
+.profile-page {
+  background-color: var(--light-yellow);
+}
+
+.profile-wrapper {
+  position: relative;
+  height: 80vh;
+}
+
+.profile-picture {
+  height: 200px;
+  position: absolute;
+  left: 5px;
+  top: 39.5px;
+}
+
+.profile-bubble {
+  position: absolute;
+  left: 150px;
+  top: 25.67px;
+  height: 200px;
+}
+
+.profile-bio {
+  position: absolute;
+  top: 220.9px;
+  left: -15px;
+  height: 170px;
+  z-index: 1;
+}
```

Comment:
There might be issues, if someone puts in loads of text, so maybe height should be auto